powersimdata/tests/mock_grid.py

```diff
@@ -14,7 +14,7 @@
 }
 
 gencost_names = {"gencost_before": "before", "gencost_after": "after"}
-storage_names = {"storage_gen": "gen"}
+storage_names = {"storage_gen": "gen", "storage_StorageData": "StorageData"}
```

Comment:
Shall we use "storage_data" instead of "storage_StorageData" or there is another `storage_data` defined somewhere else.

Reply:
The logic here was: `"storage_gen"` reflects the `"gen"` key in the `storage` dict. Therefore, `"storage_StorageData"` reflects the `"StorageData"` key in the `storage` dict. But it is arbitrary, I don't mind changing it to whatever you prefer.

There's a longer term project here to update our storage schema to something more intuitive (now that we don't need to maintain compatibility with Matpower), but that will require deeper design thinking.

Reply:
I see. Either way. No strong preference ai this point.